src/_help/organizations/create-and-manage-organizations.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Create and manage organizations
+---
+
+- TOC
+{:toc}
+
+## How to create an organization
+To create an organization, you should start by accessing the settings of your personal account.
+The Organizations section allows you to view the organizations you are currently associated with and create a new one.
+
+![](/images/help/create-org.png)
+
+You will then be prompted to specify the organization's name (visible to all its members and guests) and its slug, which determines its URL (in the format bump.sh/your-organization-slug/).
+
+![](/images/help/org-creation.png)
+
+At this stage, you can also choose to add existing documentation and/or hubs to your new organization. These will retain the access settings they had before the migration.
+
+## Add teammates to an organization
+From the Members section in the settings of an organization, you can add new members, view the list of existing members, modify their roles, and check pending invitations.
+
+To invite a new member, simply provide their email address and specify the role you wish to assign to them.
+
+![](/images/help/org-add-member.png)
+
+[Roles](/help/organizations/organization-access-management/#roles) allow you to precisely adjust what a member of your organization can or cannot do. You can change a member's role at any time.
+
+> Please note that only the owner of the organization has access to billing and its options.
+{: .info}
+
+### Add teammates with SSO
+The SSO support allows you to add and manage members of your organization directly through your existing solution. We support most of the available identity providers (list them), through WorkOS.
+
+> Using SSO does not prevent you from inviting users via manual invitations.
+{: .info}
+
+> SSO is a feature available from our Business plan.
+{: .info}
+
+## Delete teammates in an organization
+Still from the Members section, you can easily remove a member from your organization by clicking on their role to the right of their name and email. The option to delete this member is available at the very end of the list that appears.
+
+## Change URL/slug
+From the settings of your organization, you can quickly change the slug via the dedicated field.
+
+Bump.sh does not automatically redirect to the new URL after changing the slug, so be sure to notify your users of this change.
+
+The Custom Domain option allows you to keep your users within your brand experience by fully customizing the URL of your organization.
+
+> Modifying the slug also changes all the public URLs for accessing your organization, hubs, and documentation!
+{: .warning}
+
+## Transfer Ownership
+Transferring ownership is mandatory if you want to leave an organization. You can only transfer ownership to a member with admin role.
+You can transfer the ownership of your organization by contacting us directly.
```

Comment:
I think te sentence could be a bit easier to understand by starting by the goal. Something like "As the owner of an organization, before leaving this one you need to transfer your ownership to another member. This can be done by contacting us."

Reply:
Is leaving the organization the only reason you want to transfer the ownership?